data/google-chrome.desktop

```diff
@@ -105,6 +105,14 @@ Comment[vi]=Truy cập Internet
 Comment[zh_CN]=访问互联网
 Comment[zh_HK]=連線到網際網路
 Comment[zh_TW]=連線到網際網路
+# Hide this desktop file if the Endless-provided Chrome flatpak is not installed.
+# This removes the feature (available in EOS <4.0) which always shows the Chrome
+# icon, and installs Chrome on demand if it’s clicked. Now, the icon will only
+# be shown if the Endless Chrome flatpak is installed. If the upstream (flathub)
```

Comment:
Does installing and removing the Flatpak cause GNOME Shell to reevaluate all `TryExec` lines in existing `.desktop` files?

My belief is yes, `GAppInfoMonitor` will emit `::changed`, shell-app-cache.c (with some delay & a thread) will call `g_app_info_get_all()`, and GLib internally re-enumerates all the desktop files.